src/main/java/uk/gov/hmcts/divorce/common/model/State.java

```diff
@@ -60,6 +60,12 @@
             if (hasAwaitingDocuments(caseData)) {
                 errors.add("Awaiting documents");
             }
+            if (!caseData.hasStatementOfTruth()) {
```

Comment:
surely these things are either or - in that either the citizen needs to have done the SOT or the solicitor needs to have. Probably not both

Reply:
My misunderstanding - have updated